lib/dependency-manager-adapters/npm.js

```diff
@@ -7,13 +7,37 @@ var path = require('path');
 var extend = require('extend');
 var debug = require('debug')('ember-try:dependency-manager-adapter:npm');
 var rimraf = RSVP.denodeify(require('rimraf'));
+var execFileSync = require('child_process').execFileSync;
 
 module.exports = CoreObject.extend({
   init: function() {
     this._super.apply(this, arguments);
     this.run = this.run || require('../utils/run');
+    this.adapterInit();
   },
-  configKey: 'npm',
+  adapterInit() {
+    this.yarnLockPresent = fs.existsSync(path.join(this.cwd, this.yarnLock));
+    let yarnManagerPresent;
+    try {
+      execFileSync('yarn', [].concat(['--version']));
+      yarnManagerPresent = true;
+    } catch (e) {
+      yarnManagerPresent = false;
+    }
+    if (this.yarnLockPresent || yarnManagerPresent) {
+      this.useYarn = true;
+      this.configKey = 'yarn';
```

Comment:
That's true. I seem to have missed that part completely. 

The only question left here is which package manager do we display in the results table. Do we always display `npm` or use `yarn` - `npm` depending on which one was used. I prefer the later option.

@rwjblue what is your point of view on that question?